src/test/java/com/amihaiemil/charles/aws/AwsHttpHeadersTestCase.java

```diff
@@ -56,23 +56,33 @@ public void fetchesOriginalRequest() {
     
     /**
      * AwsHttpHeaders can perform the original {@link AwsHttpRequest}
+     */
+    @Test
+    public void performsRequest() {
+        AwsHttpHeaders<String> awsh = new AwsHttpHeaders<>(
+            new AwsHttpRequest.FakeAwsHttpRequest(),
+            new HashMap<String, String>()
+        );        
+        assertTrue(awsh.perform().equals("performed fake request"));
```

Comment:
@SherifWaly As you see in [this](https://github.com/opencharles/charles-rest/issues/178) issue, we want to start using hamcrest matchers instead of simple asserts (see there the explanation).

Can you change the asserts to matcher asserts? It's not hard at all and the gain is big. See [this](https://github.com/decorators-squad/camel/blob/master/src/test/java/com/amihaiemil/camel/ScalarTest.java) test class for examples.

Any questions, please ask.